src/snovault/tests/elasticsearch_fixture.py

```diff
@@ -45,6 +46,15 @@ def server_process(datadir, host='127.0.0.1', port=9201, prefix='', echo=False):
         msg = ('Process return code: %d\n' % code) + b''.join(lines).decode('utf-8')
         raise Exception(msg)
 
+    es = Elasticsearch('%s:%s' % (host, port))
+    es.indices.put_template(name="default", order=0, body={
+        "index_patterns": ["*"],
+        "settings": {
+            "number_of_shards": "1",
+            "number_of_replicas": "0",
+        },
+    })
+
```

Comment:
This template doesn't do anything because they end up overridden by the per index settings in create_mapping. Doesn't seem to make much difference when running locally. (I don't think number_of_replicas has any effect in a single node cluster.) Not sure what to do to make tests run in Travis at this point (they seem to just get stuck.) May be worth exploring other options.